arithmetization/src/test/java/net/consensys/linea/zktracer/module/ecdata/EcRecoverTest.java

```diff
@@ -60,6 +83,35 @@ void testEcRecover(
       boolean expectedInternalChecksPassed,
       boolean expectedSuccessBit,
       TestInfo testInfo) {
+    testEcRecoverBody(
+        description, h, v, r, s, expectedInternalChecksPassed, expectedSuccessBit, testInfo);
+  }
+
+  @Tag("nightly")
+  @ParameterizedTest
+  @MethodSource("ecRecoverSourceNightly")
```

Comment:
So we have the same exact test twice ? Once without a tag, once with the "nightly" tag ?

Reply:
Ah, the sources are different. OK.